src/user/user.controller.ts

```diff
@@ -34,13 +36,28 @@ export class UserController {
     return req['user'];
   }
 
+  // TODO - work out if this is used anywhere and delete if necessary
   @ApiBearerAuth()
   @Post('/delete')
   @UseGuards(FirebaseAuthGuard)
+  async deleteUserRecord(@Req() req: Request): Promise<string> {
```

Comment:
Heya - how come you've added `deleteUserRecord` here as it seems to do the same thing as `deleteUser`